src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -696,33 +696,37 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref HashS
 
             static TypeSymbol? visitFunctionPointerType(FunctionPointerTypeSymbol type, Func<TypeWithAnnotations, T, bool, bool>? typeWithAnnotationsPredicate, Func<TypeSymbol, T, bool, bool>? typePredicate, T arg, bool useDefaultType, bool canDigThroughNullable)
             {
-
                 MethodSymbol currentPointer = type.Signature;
-                var result = visitType(currentPointer.ReturnTypeWithAnnotations);
+                var result = VisitType(
+                    typeWithAnnotationsOpt: canDigThroughNullable ? default : currentPointer.ReturnTypeWithAnnotations,
+                    type: canDigThroughNullable ? currentPointer.ReturnTypeWithAnnotations.NullableUnderlyingTypeOrSelf : null,
```

Comment:
>currentPointer.ReturnTypeWithAnnotations [](start = 50, length = 40)

consider extracting local. Same for `parameter.TypeWithAnnotations`

It looks like this is just inlining the old code. Or did I miss some difference?

Reply:
This is feedback from the previous PR.

---
In reply to: [422348522](https://github.com/dotnet/roslyn/pull/44078#discussion_r422348522) [](ancestors = 422348522)

Reply:
So it is just inlining, or there is some difference? I'm not casting judgement, I've starred at the diff and want to make sure I understood :-)

---
In reply to: [422419109](https://github.com/dotnet/roslyn/pull/44078#discussion_r422419109) [](ancestors = 422419109,422348522)

Reply:
> So it is just inlining, 

The motivation is to reduce used stack size by eliminating a capture and unnecessary stack frames.

---
In reply to: [422422459](https://github.com/dotnet/roslyn/pull/44078#discussion_r422422459) [](ancestors = 422422459,422419109,422348522)